tests/integration/README.md

```diff
@@ -1,25 +1,18 @@
 ## Integration Testing
+Integration test verifies the functionality of EBS CSI driver as a standalone server outside kuberentes. It exercises the lifecycle of create, attach, stage, mount volume and the reverse operations. And it verifies data can be written onto an EBS volume without any issue.
 
-### Requirements 
+## Run Integration Tests Locally
+Integration test is executed using [aws-k8s-tester](https://github.com/aws/aws-k8s-tester) which is CLI tool for k8s testing on AWS. With aws-k8s-tester, it automates the process of provisioning EC2 instance, pull down and build EBS CSI driver, running the defined integration test and sending test result back. See aws-k8s-tester for more details about how to use it.
 
-1. macOS or Linux
-1. `GOPATH` environment variable [set](https://github.com/golang/go/wiki/SettingGOPATH)
-1. [AWS account](https://aws.amazon.com/account/) that has been [configured locally](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+### Requirements
+1. AWS credential is [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+1. Other requirements needed by aws-k8s-tester
```

Comment:
Maybe a link for `aws-k8s-tester` requirements would be nice (if there's any)

Reply:
There isn't any yet. Created an issue to track this: https://github.com/aws/aws-k8s-tester/issues/32